net/nimble/host/include/host/ble_store.h

```diff
@@ -31,6 +31,12 @@ extern "C" {
 #define BLE_STORE_OBJ_TYPE_PEER_SEC     2
 #define BLE_STORE_OBJ_TYPE_CCCD         3
 
+/** Failed to persist record; insufficient storage capacity. */
+#define BLE_STORE_EVENT_OVERFLOW        1
+
+/** About to execute a procedure that may fail due to overflow. */
+#define BLE_STORE_EVENT_OVERFLOW_NEXT   2
```

Comment:
good point. somehow I forgot about that obj_type. So callback first checks event_type then looks into obj_type and then specific data.  You are right.. in this case it makes sense to have generic event_type.

So maybe we can try to improve name: e.g.  BLE_STORE_EVENT_OVERFLOW_WARNING, BLE_STORE_EVENT_OVERFLOW_IS_COMING, BLE_STORE_EVENT_OVERFLOW_AHEAD, BLE_STORE_EVENT_POSSIBLE_OVERFLOW ? Not sure if any of this is better :)